androbot/actions.py

```diff
@@ -0,0 +1,81 @@
+import random
+
+from loguru import logger
+
+from . import crud, schemas
+from .config import settings
+from .crud import get_question, is_tg_user_already_exist
+from .database import SessionLocal
+from .errors import UserExistsException, UserNotExistsException
+from .models import Question
+from .specialty import Specialty
+
+
+def get_main_menu():
```

Comment:
```suggestion
def get_main_menu() -> List[str]:
```
Типы :)
Тут и дальше